app/views/complaints/_complaints_data.html.erb

```diff
@@ -0,0 +1,53 @@
+<% if @complaints.length == 0 %>
```

Comment:
should the alert be its own partial? i thought that might be overly complex.

Reply:
[suggestion] I do think it would clean it up to have the "with data" and "no data" versions be their own files. I'd do the switch in `index.html` though so this partial doesn't have any `if/else` at all.

Reply:
agreed!!!! thanks for the push